swupd/swupd.go

```diff
@@ -173,9 +252,10 @@ func (s *SoftwareUpdater) Verify(version string, mirror string, verifyOnly bool)
 
 // VerifyWithBundles runs "swupd verify" operation with all bundles
 func (s *SoftwareUpdater) VerifyWithBundles(version string, mirror string, bundles []string) error {
+	// the "swupd verify" command is being deprecated, use "swupd os-install" instead
 	args := []string{
 		"swupd",
-		"verify",
+		"os-install",
```

Comment:
I think it's better to leave verify for now on installer and just change it to use os-install when swupd is released with that feature.
If we merge that on installer it won't work with last released version of swupd.

Reply:
well, this change cannot be merged in the clr-installer anyway until swupd releases, because it requires some other commits that are in master like the JSON output from swupd. If that is the case the os-install command will be released by then. Unless we are planning on making functionality like the JSON output available sooner than the os-install, like in a different, earlier release.